olp-cpp-sdk-dataservice-read/test/unit/src/CatalogClientTest.cpp

```diff
@@ -940,6 +941,7 @@ class CatalogClientMockTest : public CatalogClientTestBase {
 
     // Catch any non-interesting network calls that don't need to be verified
     EXPECT_CALL(*network_mock_, Send(_, _, _, _, _)).Times(testing::AtLeast(0));
+    EXPECT_CALL(*network_mock_, Cancel(_)).Times(testing::AtLeast(0));
```

Comment:
```suggestion
    EXPECT_CALL(*network_mock_, Cancel(_)).Times(0);
```
Or as specified before, even Google is recommending this, use StrictMock to guard against unexpected calls:

```
[INFO] StreamLayerClientImpl - Flushed 1 publish requests

GMOCK WARNING:
Uninteresting mock function call - returning directly.
    Function call: Cancel(5)
NOTE: You can safely ignore the above warning unless this call should not happen.  Do not suppress it by blindly adding an EXPECT_CALL() if you don't mean to enforce the call.  See https://github.com/google/googletest/blob/master/googlemock/docs/CookBook.md#knowing-when-to-expect for details.
```